webserver/__init__.py

```diff
@@ -74,4 +74,10 @@ def create_app():
     app.register_blueprint(user_bp)
     app.register_blueprint(datasets_bp, url_prefix='/datasets')
 
+    # Admin section
+    from flask_admin import Admin
+    from webserver.admin import views as admin_views
```

Comment:
Yes, but it requires overriding built-in templates, which I don't want to do too much. From the [docs](https://flask-admin.readthedocs.org/en/latest/introduction/#overriding-the-built-in-templates):
> To take full control over the style and layout of the admin interface, you can override all of the built-in templates. Just keep in mind that the templates will change slightly from one version of Flask-Admin to the next, so once you start overriding them, you need to take care when upgrading your package version.

Besides this is not meant for public view.